src/components/implementation/tests/micro_booter/mb_tests.c

```diff
@@ -468,10 +413,11 @@ test_budgets_single(void)
 
 	PRINTC("Budget switch latencies: ");
 	for (i = 1 ; i < 10 ; i++) {
```

Comment:
This has been there, I didn't work on fixing hard-coded things in `micro_booter`. Changes here are specific to `cos_rcv/cos_sched_rcv` api changes.